src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="?"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label>
+        <input
+          type="text"
+          class="input_text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+        <img
+          src="/images/Search.svg"
+          alt="search_icon"
+          class="search_icon"
+        >
```

Comment:
Better to use background image for decorative icon

Reply:
done.